boards/xtensa/intel_adsp_cavs25/doc/intel_adsp_generic.rst

```diff
@@ -0,0 +1,342 @@
+:orphan:
+
+.. _intel_adsp_generic:
+
+Intel ADSP CAVS and ACE
+#######################
+
+Intel's Audio and Digital Signal Processing (ADSP) product offerings include
+the Converged Audio Video Sensing (CAVS) IP and its successor, the
+Audio and Context Engine (ACE). The open-source version of the IP's firmware
+is known as Sound Open Firmware (`SOF`_). It can be built with either Zephyr or
+Cadence's proprietary Xtensa OS (XTOS) and run on several Intel ('Lake') and
+non-Intel platforms. Below is a mapping of firmware versions to platforms.
+
++----------+--------------+
+| Firmware | Platform     |
++==========+==============+
+| CAVS 1.5 | Apollo Lake  |
++----------+--------------+
+| CAVS 1.8 | Whiskey Lake |
++----------+--------------+
+| CAVS 2.5 | Tiger Lake   |
++----------+--------------+
+| ACE 1.5  | Meteor Lake  |
++----------+--------------+
+
+The boards in the Zephyr tree are named after the version of the firmware that
```

Comment:
So here too, it's not the firmware version but version of the audio DSP hardware. Otherwise table is correct.